utils/biospecimen.js

```diff
@@ -1,5 +1,6 @@
 const { getResponseJSON, setHeaders, logIPAdddress, SSOValidation, convertSiteLoginToNumber } = require('./shared');
 const fieldMapping = require('./fieldToConceptIdMapping');
+const { sendInstantNotification } = require("./notifications");
```

Comment:
should we only have this `require` statement inside the `else if` loop where we are using it so that we aren't unnecessarily requiring it when we don't use it?

Reply:
This is something we need to discuss, and maybe some tests are needed to find out which method of import is faster.
My understanding is that if an important is used inside an API, the import is run each time the API is called. For APIs frequently called, putting imports at module level may have performance advantage.